### PR TITLE
E2E / ETH TX test improvements

### LINKED ISSUE #939: Complete send eth E2E test
The test `test/e2e/ethtransfer_test.go` is incomplete, it's lacking:

- `opsman` support to spin up the environment
- asserts to check that the batches get virtualized
- asserts to check that the batches get consolidated
- to be added in the tests run in GHA

### PR DESCRIPTION
**Pull request by KonradIT**
Closes #939.

### What does this PR do?

Completes the E2E ETH TX (send 1000 Transactions) Test

### Reviewers

Main reviewers:

- @arnaubennassar 
- @tclemos 
- @ToniRamirezM 

Codeowner reviewers:

### CODE REVIEW

**Comment by KonradIT** [comment]
Add test: check every TX Hash for: `WaitTxtToBeMined`, + `L2BlockNum == Nonce`

**Comment by KonradIT** [comment]
@tclemos @arnaubennassar What do you think about adding the commands for `make run-db` and `make run-zkprover` at the start using `StartComponent`, before setup?

**Comment by KonradIT** [comment]
Next run will fail! This is expected, I added a test which fails for now.

```
e2e/ethtransfer_test.go:56	Sending 100 transactions...
    /home/konrad/hermez/core/test/e2e/ethtransfer_test.go:89:
        	Error Trace:	/home/konrad/hermez/core/test/e2e/ethtransfer_test.go:89
        	Error:      	Not equal:
        	            	expected: 0x1
        	            	actual  : 0x64
        	Test:       	Test1000EthTransfer
```

File: jsonrpc/interfaces.go [hunk 1]
```diff
@@ -53,8 +53,8 @@ type stateInterface interface {
 	GetL2BlockHashesSince(ctx context.Context, since time.Time, dbTx pgx.Tx) ([]common.Hash, error)
 	DebugTransaction(ctx context.Context, transactionHash common.Hash, tracer string) (*runtime.ExecutionResult, error)
 	ProcessUnsignedTransaction(ctx context.Context, tx *types.Transaction, senderAddress common.Address, blockNumber uint64, dbTx pgx.Tx) *runtime.ExecutionResult
-	IsBatchConsolidated(ctx context.Context, batchNumber int, dbTx pgx.Tx) (bool, error)
-	IsBatchVirtualized(ctx context.Context, batchNumber int, dbTx pgx.Tx) (bool, error)
+	IsL2BlockConsolidated(ctx context.Context, batchNumber int, dbTx pgx.Tx) (bool, error)
+	IsL2BlockVirtualized(ctx context.Context, batchNumber int, dbTx pgx.Tx) (bool, error)
```
Comment (tclemos) [inline jsonrpc/interfaces.go:57]: batchNumber = blockNumber

File: jsonrpc/zkevm.go [hunk 2]
```diff
@@ -29,31 +29,31 @@ func (h *ZKEVM) ConsolidatedBlockNumber() (interface{}, rpcError) {
 	})
 }
 
-// IsBatchConsolidated returns the consolidation status of a provided batch ID
-func (h *ZKEVM) IsBatchConsolidated(batchNumber int) (interface{}, rpcError) {
+// IsL2BlockConsolidated returns the consolidation status of a provided batch ID
+func (h *ZKEVM) IsL2BlockConsolidated(batchNumber int) (interface{}, rpcError) {
```
Comment (tclemos) [inline jsonrpc/zkevm.go:33]: batchNumber = blockNumber

File: jsonrpc/zkevm.go [hunk 3]
```diff
@@ -29,31 +29,31 @@ func (h *ZKEVM) ConsolidatedBlockNumber() (interface{}, rpcError) {
 	})
 }
 
-// IsBatchConsolidated returns the consolidation status of a provided batch ID
-func (h *ZKEVM) IsBatchConsolidated(batchNumber int) (interface{}, rpcError) {
+// IsL2BlockConsolidated returns the consolidation status of a provided batch ID
```
Comment (tclemos) [inline jsonrpc/zkevm.go:32]: batch ID = block number

File: jsonrpc/zkevm.go [hunk 4]
```diff
@@ -29,31 +29,31 @@ func (h *ZKEVM) ConsolidatedBlockNumber() (interface{}, rpcError) {
...
 		}
 
-		return isBatchConsolidated, nil
+		return IsL2BlockConsolidated, nil
 	})
 }
 
-// IsBatchVirtualized returns the virtualisation status of a provided batch ID
-func (h *ZKEVM) IsBatchVirtualized(batchNumber int) (interface{}, rpcError) {
+// IsL2BlockVirtualized returns the virtualisation status of a provided batch ID
```
Comment (tclemos) [inline jsonrpc/zkevm.go:46]: batch ID = block number

File: jsonrpc/zkevm.go [hunk 5]
```diff
@@ -29,31 +29,31 @@ func (h *ZKEVM) ConsolidatedBlockNumber() (interface{}, rpcError) {
...
+func (h *ZKEVM) IsL2BlockVirtualized(batchNumber int) (interface{}, rpcError) {
```
Comment (tclemos) [inline jsonrpc/zkevm.go:47]: batchNumber = blockNumber

**Review by tclemos — APPROVED** [review]
LGTM, just need to fix some names

**Comment by KonradIT** [comment]
@tclemos The `test/e2e/ethtransfer_test.go` is not symlinked to `ci/e2e-groupX` so its not running on GHA for now.

**Review by ToniRamirezM — APPROVED** [review]

**Review by arnaubennassar — APPROVED** [review]

**Pull request merged**